### PR TITLE
User Profile with Animation

### PR DESCRIPTION
**Pull request by tnptw**

### CODE REVIEW

File: src/pages/account/profile.tsx [hunk 1]
```diff
@@ -14,19 +15,47 @@ const profile = () => {
...
-        <div className="p-4">{JSON.stringify(profile)}</div>
-        <button
-          className="p-4 border-2 border-red-500 hover:bg-red-500"
-          onClick={handleLogout}
-        >
-          Logout
-        </button>
+    <>
+      <NavBar />
+      <div className="flex bg-blue h-screen justify-center">
```
Comment (nnanthicha) [inline src/pages/account/profile.tsx:22]: put "bg-blue h-screen" before navbar ?

**Review by nnanthicha — CHANGES_REQUESTED** [review]

File: src/pages/account/profile.tsx [hunk 2]
```diff
@@ -14,19 +15,47 @@ const profile = () => {
...
+            <img src="../trainder_icon.png" alt="" className="w-1/3 mx-auto" />
+          </div>
+          <div className=" container bg-white p-10 rounded-3xl w-4/5 mx-auto">
+            <div className="columns-2 text-xl">
+              <div>
+                ID: <strong>*********{profile?.citizenId.slice(9, 13)}</strong>
+              </div>
+              <div>
+                Birth-Date:{" "}
+                <strong>{profile?.birthdate.toString().split("T")[0]}</strong>
```
Comment (noppakorn) [inline src/pages/account/profile.tsx:39]: can you also show date in a more beautiful manner?

**Review by noppakorn — CHANGES_REQUESTED** [review]

**Review by nnanthicha — APPROVED** [review]

File: src/pages/account/profile.tsx [hunk 3]
```diff
@@ -14,20 +15,112 @@ const profile = () => {
...
+    const year = profile?.birthdate.toString().split("T")[0].split("-")[0];
+    return (
+      <div>
+        Birth:{" "}
+        <strong>
+          {day}/{month_str}/{year}
+        </strong>
+      </div>
+    );
+  };
```
Comment (noppakorn) [inline src/pages/account/profile.tsx:45]: Why are you doing this? In `UserProfile` the birthdate is a `Date` object right? Couldn't you just use the built-in method? [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)

File: src/pages/account/profile.tsx [hunk 4]
```diff
@@ -14,20 +15,112 @@ const profile = () => {
...
+  const getGender = () => {
+    const gender_first = profile?.gender[0].toUpperCase();
```
Comment (noppakorn) [inline src/pages/account/profile.tsx:47]: prefer camelCase

**Review by noppakorn — CHANGES_REQUESTED** [review]

**Review by noppakorn — APPROVED** [review]

**Pull request merged**